test/trento/application/usecases/clusters_test.exs

```diff
@@ -30,6 +33,47 @@ defmodule Trento.ClustersTest do
     end
   end
 
+  describe "checks execution with wanda adapter" do
```

Comment:
What about moving this tests in its separate module, Wanda related?

At the end what we are testing here is the behavior in wanda scenario
```elixir
assert :ok = Clusters.Wanda.request_checks_execution(cluster_id)
```

Reply:
I just added the wanda word in the text to make differentiation easy. At some point the old runner code will be removed, and we won't have such a differentiation (neither this adapter/behaviour code), so most probably wanda terminology will go away from here, because it will simply be how checks execution is requested. And I would be simple to know that these tests must not be deleted (and we can rename the describe text as well).
What do you think?

Reply:
Alright, makes sense, thanks!